src/librustc_mir/interpret/mod.rs

```diff
@@ -1,5 +1,10 @@
 //! An interpreter for MIR used in CTFE and by miri
 
+#[macro_export]
+macro_rules! err {
+    ($($tt:tt)*) => { Err($crate::interpret::EvalErrorKind::$($tt)*.into()) };
+}
```

Comment:
macro ordering is important. If we move it to `error.rs`, we need to ensure that the `mod error` is the first one in the list, otherwise the modules above don't get the macro.